addon/components/frost-link.js

```diff
@@ -309,6 +325,18 @@ export default LinkComponent.extend(PropTypeMixin, HookMixin, SpreadMixin, {
   },
 
   /**
+   * Open a link for an external location.
+   */
+  _openExternalLink () {
+    const href = this.get('href')
+    // If we want to open a new tab, the target will be set accordingly when we setup the routing.
+    // If we want to stay in the same tab, we will set the href.
+    if (this._shouldOpenInSameTab()) {
+      window.location.href = href
```

Comment:
Doesn't implementing it this way prevent the user from choosing, at the time of clicking the link, whether to open it in the same tab or in another tab? I'm not a big fan of links that redirect with JavaScript vs actually setting an `href` on an anchor tag. The latter is much more user friendly IMO. Instead of putting the decision of whether to open in the same or a new tab solely in the hands of the developer, it lets the user have some control over it. For example, I believe the UX on the links within an object browser are supposed to open in the same tab, but allow the user to open them in a new tab if desired. I don't think this implementation will allow that use-case.